package.json

```diff
@@ -10,10 +10,14 @@
   "private": true,
   "dependencies": {
     "classnames": "^2.2.6",
+    "moment": "^2.24.0",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
+    "react-datetime": "^2.16.3",
+    "react-day-picker": "^7.3.2",
```

Comment:
I think we should only add to dependencies `moment` and `react-datetime`. 2 other added libs should be removed from `package.json` and yarn lock file. Docs of `yarn remove` command: https://yarnpkg.com/en/docs/cli/remove

Reply:
sure